optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/persistence/ConferenceSchedulingCfpDevoxxImporter.java

```diff
@@ -415,9 +412,9 @@ private void importTimeslotList() {
                     startAndEndTimeToTimeslotMap.put(Pair.of(startDateTime, endDateTime), timeslot);
                 }
 
-                if (!timeslotObject.isNull("talk")) {
-                    scheduleTalk(timeslotObject, room, timeslot);
-                }
+//                if (!timeslotObject.isNull("talk")) {
+//                    scheduleTalk(timeslotObject, room, timeslot);
+//                }
```

Comment:
Is this intended?

If it's unuseful code, its usually better to delete it

Reply:
No it's not intended! I commented it out to test importing, because the REST API still have some inconsistencies, thanks for pointing it out